cgmanifest.json

```diff
@@ -21543,8 +21543,8 @@
         "type": "other",
         "other": {
           "name": "PyGreSQL",
-          "version": "5.2.2",
-          "downloadUrl": "https://github.com/PyGreSQL/PyGreSQL/archive/5.2.2/PyGreSQL-5.2.2.tar.gz"
+          "version": "6.0.1",
+          "downloadUrl": "https://github.com/PyGreSQL/PyGreSQL/archive/refs/tags/6.0.1.tar.gz"
```

Comment:
Upload to blobstore: [link](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=649238&view=results)